src/Microsoft.DotNet.Configurer/BashPathUnderHomeDirectory.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+
+namespace Microsoft.DotNet.Configurer
+{
+    public struct BashPathUnderHomeDirectory
```

Comment:
GetHomeDirectroyWithTilda,
GetHomeDirectroyWithDollar,
GetFullHomeDirectory

is getting too funny. So I made a struct out